src/Data.php

```diff
@@ -112,6 +119,30 @@ public static function setFallbackLocale($locale)
         }
     }
 
+    /**
+     * Get the data root directory.
+     *
+     * @return string
+     */
+    public static function getDataDirectory()
+    {
+        if (!isset(static::$directory)) {
+            static::$directory = __DIR__.DIRECTORY_SEPARATOR.'data';
+        }
+
+        return static::$directory;
+    }
+
+    /**
+     * Set the data root directory.
+     *
+     * @param string $directory
+     */
+    public static function setDataDirectory($directory)
+    {
+        static::$directory = $directory;
```

Comment:
What about clearing the cache (`static::$cache[] = array();`)? That way, we avoid returning data from the previous data directory,

Reply:
Good idea :-) Fixed.